File: dmoj.ca-downloaded-submissions/CCC/ccc08s2.cpp
#include <stdio.h>
#include <math.h>

int main(){
    int r;
    for(scanf("%d", &r); r != 0; scanf("%d", &r)){
        int total = r;
        for(int row = 1; row <= r; row++){
            total += sqrt((r-row)*(r+row));
        }
        printf("%d\n", total*4+1);
    }
    return 0;
}

File: dmoj.ca-downloaded-submissions/CCC/ccc08s2.py
from math import pi
import math

r = int(input())
while(r != 0):
    right = r
    total = r
    for row in range(1, r+1):
        while row**2+right**2 > r*r:
            right -= 1
        total += right
    print(total*4 + 1)
    r = int(input())